src/tests/CMakeLists.txt

```diff
@@ -142,12 +139,11 @@ endif()
 
 if( MFEM_FOUND )
   set(mfem_smoke_depends mfem gtest)
-  blt_list_append(TO mfem_smoke_depends ELEMENTS umpire cuda IF ENABLE_CUDA )
   blt_add_executable(
     NAME mfem_smoke_test
     SOURCES mfem_smoke.cpp
     OUTPUT_DIR ${TEST_OUTPUT_DIRECTORY}
-    DEPENDS_ON ${mfem_smoke_depends}
+    DEPENDS_ON ${mfem_smoke_depends} ${tribol_device_depends}
```

Comment:
This is a pretty easy thing to forget.. i would hide it in a macro but up to you.